src/plugins/kibana_usage_collection/server/collectors/core/core_usage_collector.ts

```diff
@@ -7,7 +7,8 @@
  */
 
 import { UsageCollectionSetup } from '@kbn/usage-collection-plugin/server';
-import { CoreUsageData, CoreUsageDataStart } from '@kbn/core/server';
+import { CoreUsageDataStart } from '@kbn/core/server';
+import { CoreUsageData } from './core_usage_data';
```

Comment:
Duplicating the types used by the collector until https://github.com/elastic/kibana/issues/139389 is resolved to avoid blocking  IDM